### DISCUSSION TITLE
Student Developer Pack - Change account receiving benefits

### DISCUSSION THREAD

**Discussion by ghost** [community/community#127299] [GitHub Education]
### Hi everyone,

Back when I first applied for the student pack, I chose to apply with a brand new account exclusively for school. However, now I realize I should've used my personal account instead, as I'd like to own and have access to my repositories after my school email expires when I graduate. Would it be possible to 'cancel' the student pack and re-apply from my personal account? Or, if not possible, to simply not renew the benefits once they expire and apply from the other account?

**Comment by davevad93** [comment]
Hi @A01643020 ,

The thing is that you need an educational email to apply for student benefits so If you're going to graduate soon and consequently lose your educational email, apart from not being able to access your other GitHub account, you won't be able to get the Student Developer Pack again either. The only thing I can suggest you to do in order to not lose access to your school's account is to add another email and verify it so when your edu email expires, you'll still be able to access that account hence not losing access to your repos. However, you won't be able to reapply for your student benefits because you'll no longer have a student status.

**Reply by ghost** [reply]
Ah, no worries, I graduate in a couple of years. I guess I could add another email when the time comes. The main problem is that account switching on github as well as my local machine is very annoying, so if possible I'd just like to move all my repos to my personal account and apply for the pack from there.

**Reply by davevad93** [reply]
Hey @A01643020 , yes switching accounts can be very annoying, however I don't think you can reapply from you personal account using the same edu email because you will surely get a message like "Academic email address already used" as stated in the [docs](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-education-for-students/why-wasnt-my-application-to-github-education-for-students-approved).
